cli/src/commands/util.rs

```diff
@@ -34,31 +35,27 @@ pub(crate) enum UtilCommand {
 }
 
 /// Print a command-line-completion script
+///
+/// Apply it by running this:
+/// bash: source <(jj util completion)
+/// fish: jj util completion --fish | source
+/// zsh:
+/// autoload -U compinit
+/// compinit
+/// source <(jj util completion --zsh)
```

Comment:
i wasn't able to figure out how to get these working with powershell and elvish. `Invoke-Command` *looks* like the right thing for powershell, but when i try `Invoke-Expression [String]$(cargo r -- util completion powershell)` it complains that it doesn't like the `using` token. i'm not sure if that's specific to powershell on windows or something like that?

elvish i couldn't get working at all, it complained about invalid syntax. i think ubuntu might package an ancient version or something like that.